pkg/apis/extensions/v1alpha1/types_infrastructure.go

```diff
@@ -83,4 +83,20 @@ type InfrastructureStatus struct {
 	// IPs may not be stable in which case the extension controller may opt to not populate this field.
 	// +optional
 	EgressCIDRs []string `json:"egressCIDRs,omitempty"`
+	// Networking is a structure containing information about the node, service and pod network ranges.
```

Comment:
```suggestion
	// Networking contains information about cluster networking such as CIDRs.
```
Minor nit suggestion, to keep it in line with the shoot api comment.

Reply:
Corrected as requested in https://github.com/gardener/gardener/pull/9998/commits/9ace55a272ca95d90569026bc6cbfe64a2b9478f.